### PR TITLE
Uk/jira/LOK-2830: When update Citus adminUser, lokahi services not works properly.

### PR DESCRIPTION
**Pull request by Umairk95**

### CODE REVIEW

**Review by MuhammadTalhaDataQHealth — APPROVED** [review]

**Review by tahir-abb — APPROVED** [review]

File: dependencies/citus/templates/citus-worker-statefulset.yaml [hunk 1]
```diff
@@ -137,7 +139,7 @@ spec:
             command:
             - 'pg_isready'
             - '-U'
-            - 'desenv'
+            - '{{ .Values.global.citus.databaseName | default .Values.databaseName }}'
```
Comment (mershad-manesh) [inline dependencies/citus/templates/citus-worker-statefulset.yaml:142]: This be `{{ .Values.global.citus.adminUser | default .Values.Citus.adminUser }}`

the value passed to `-U` is the username

**Review by mershad-manesh — CHANGES_REQUESTED** [review]
few minor tweaks are required

File: dependencies/citus/templates/citus-worker-statefulset.yaml (hunk 1 above)
Comment (Umairk95) [reply dependencies/citus/templates/citus-worker-statefulset.yaml:146]: yes right,  Username and database both added.

File: dependencies/citus/templates/citus-worker-statefulset.yaml [hunk 2]
```diff
@@ -128,7 +130,7 @@ spec:
...
```
[the remark this answers is not in the document]
Comment (Umairk95) [reply dependencies/citus/templates/citus-worker-statefulset.yaml:135]: yes right,  Username and database both added

**Comment by mershad-manesh** [comment]
In Lokahi `values.yaml` file we have
```
global:
 ...
  citus:            # Override citus default port, database name, adminUser and adminPassword in lokahi globally
    port: ~
    adminUser: ~
    adminPassword: ~
    databaseName: ~
...
# Enables integration of Citus with notification module.
Citus:
  enabled: true
  adminUser: desenv
  adminPassword: notset
  port: 5432
```

What happens if I set  different values for these properties? Which one has the higher priority ?

**Comment by Umairk95** [comment]
> In Lokahi `values.yaml` file we have
> 
> ```
> global:
>  ...
>   citus:            # Override citus default port, database name, adminUser and adminPassword in lokahi globally
>     port: ~
>     adminUser: ~
>     adminPassword: ~
>     databaseName: ~
> ...
> # Enables integration of Citus with notification module.
> Citus:
>   enabled: true
>   adminUser: desenv
>   adminPassword: notset
>   port: 5432
> ```
> 
> What happens if I set different values for these properties? Which one has the higher priority ?

**Comment by Umairk95** [comment]
> In Lokahi `values.yaml` file we have
> 
> ```
> global:
>  ...
>   citus:            # Override citus default port, database name, adminUser and adminPassword in lokahi globally
>     port: ~
>     adminUser: ~
>     adminPassword: ~
>     databaseName: ~
> ...
> # Enables integration of Citus with notification module.
> Citus:
>   enabled: true
>   adminUser: desenv
>   adminPassword: notset
>   port: 5432
> ```
> 
> The values in global have higher priority, if any value in global is set to  NIL then it will take default value from Citus values.

**Review by mershad-manesh — APPROVED** [review]
Took a quick look at the changes, they look good

**Pull request merged**